Firestore/core/test/unit/util/random_access_queue_test.cc

```diff
@@ -0,0 +1,647 @@
+/*
+ * Copyright 2017 Google LLC
```

Comment:
- 2017: Trump but no Covid
- 2021: No Trump but Covid

Not sure which one is better, but I would still lean towards 2021.

Reply:
Tough call. But I tend to agree. Changed.